src/com/google/guiceberry/junit4/GuiceBerryRule.java

```diff
@@ -16,52 +16,34 @@
 package com.google.guiceberry.junit4;
 
 import com.google.guiceberry.DefaultEnvSelector;
-import com.google.guiceberry.GuiceBerryEnvSelector;
 import com.google.guiceberry.GuiceBerry;
-import com.google.guiceberry.GuiceBerry.GuiceBerryWrapper;
+import com.google.guiceberry.GuiceBerryEnvSelector;
 import com.google.guiceberry.TestDescription;
 import com.google.inject.Module;
-
 import org.junit.rules.MethodRule;
 import org.junit.runners.model.FrameworkMethod;
 import org.junit.runners.model.Statement;
 
 /**
- * {@link GuiceBerry} adapter for JUnit4 tests.
- * 
+ * {@link GuiceBerry} adapter for JUnit4 tests, implementing {@MethodRule}.
+ *
+ * <p>This is the preferred implementation to use, as it does not depend on the client code
```

Comment:
Ok, let's just have this as:

* {@link GuiceBerry} adapter for JUnit4 tests.
*
* @see If you are looking for a {@link TestRule}, see {@link GuiceBerryTestRule}